addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3608,14 +3611,15 @@ class SnippetsMenu extends Component {
                         // (mutexed as well).
                         dragAndDropResolve();
 
-                        await this.callPostSnippetDrop($target);
-
-                        // Restore editor to its normal edition state, also
-                        // make sure the undroppable snippets are updated.
-                        this._disableUndroppableSnippets();
-                        this.options.wysiwyg.odooEditor.unbreakableStepUnactive();
-                        this.options.wysiwyg.odooEditor.historyStep();
-                        this.$el.find('.oe_snippet_thumbnail').removeClass('o_we_already_dragging');
+                        const postSnippetDropExtraActions = () => {
+                            // Restore editor to its normal edition state, also
+                            // make sure the undroppable snippets are updated.
+                            this._disableUndroppableSnippets();
+                            this.options.wysiwyg.odooEditor.unbreakableStepUnactive();
+                            this.options.wysiwyg.odooEditor.historyStep();
+                            this.$el.find('.oe_snippet_thumbnail').removeClass('o_we_already_dragging');
+                        }
+                        await this.callPostSnippetDrop($target, postSnippetDropExtraActions);
```

Comment:
Missing `;`, your linter should have told you :grimacing: 
But just pass the function as argument directly, no need to save it into a temporary variable :pray: